docs/example/response-types/index.md

```
@@ -0,0 +1,14 @@
+---
+aside: false
+outline: false
+title: vitepress-theme-openapi
+---
+
+<script setup lang="ts">
```

Markdownlint

Element: script
Inline HTML

(MD033, no-inline-html)